testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -145,6 +145,101 @@ def test_transform_universe():
     pca_test.transform(u2)
 
 
+def test_project_no_pca_run(u, pca):
+    pca_class = PCA(u, select=SELECTION)
+    with pytest.raises(ValueError):
+        func = pca_class.project_single_frame()
+
+
+def test_project_none_anchor(u, pca):
+    group = u.select_atoms('resnum 1')
+    with pytest.raises(ValueError):
+        func = pca.project_single_frame(0, group=group, anchor=None)
+
+
+def test_project_more_anchor(u, pca):
+    group = u.select_atoms('resnum 1')
+    with pytest.raises(ValueError):
+        project = pca.project_single_frame(0, group=group, anchor='backbone')
+
+
+def test_project_less_anchor(u, pca):
+    group = u.select_atoms('all')
+    with pytest.raises(ValueError):
+        project = pca.project_single_frame(0, group=group, anchor='name CB')
+
+
+def test_project_improper_anchor(u):
+    pca = PCA(u, select='name CA').run()
+    group = u.select_atoms('all')
+    with pytest.raises(ValueError):
+        project = pca.project_single_frame(0, group=group, anchor='name N')
+
+
+def test_project_compare_projections():
+    u = mda.Universe(PSF, DCD)
+    u_copy = u.copy()
+    pca = PCA(u, select=SELECTION).run()
+    project0 = pca.project_single_frame(0)
+    project1 = pca.project_single_frame(1)
+
+    u.trajectory[0]
+    coord0 = project0(u.trajectory.ts).positions.copy()
+    u.trajectory[0]
+    coord1 = project1(u.trajectory.ts).positions.copy()
+
+    with pytest.raises(AssertionError):
```

Comment:
Or do you expect this situation to fail? Then mark the test xfail. 

But please explain what this test is supposed to show.

Reply:
verifies that the projections along two different PCs are different. Added "assert not np.allclose()" because we want it to detect future changes in the code, in case it returns the same timestep (or the same projection).